less/global.less

```diff
@@ -32,7 +37,6 @@ p {
 
 .container {
   max-width: 800px;
-  width: 100%;
-  margin: 0 auto;  
+  width: 98%;
```

Comment:
great change, it seems minor but it ensures that on mobile view the content doesn't touch edges